index.html

```diff
@@ -83,76 +83,76 @@ <h4>Multi-Post Stories</h4>
         
             <div class="cards-section">
                 <div class="work-card">
-                    <div class="card-content">
-                        <h4>Professional Art Printing Data</h4>
+                    <div class="card-content flex-container " >
+                        <h4 class="title" > 1st: Printing Data</h4>
```

Comment:
https://github.com/samiullahbahadur/Portfolio/blob/d4215e1d6f2121568876273d9c2c09ebf61e81fd/index.html#L70-L158


- [ ] As required :point_down: 

![image](https://user-images.githubusercontent.com/57421823/151015393-2368bdbc-0f28-46f4-9357-b17c3811e52d.png)

- [ ] As seen above, all the projects section is created dynamically with the information from the projects array (all the HTML of the section is created when the page loads). This requires changes in existing code - already created HTML code for projects list should be removed and instead it should be added via JavaScript.

This means you can create an array of objects to store the card-content project information then populate the `work-card` section of the main page entirely with javascript!

If you are struggling or don't know how to do so, here is an [example](https://github.com/olawale-o/portfolio-mobile-version/pull/15/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R95)

 - All those cards information should be stored into an array of objects then create a function to render through the array and create dynamically your HTML with the information from the object array, the works section must stay empty

Reply:
I'm did it before, You should checked my  JavaScript code and then suggest for me, I think you checked just index page you don't checked my js code
what do you mean man?
you should connect with me by zoom and I will show you this parts.

Reply:
Hi @samiullahbahadur, 

https://github.com/samiullahbahadur/Portfolio/blob/463880b6331577c1b02b42227f5dee2503cadf54/js/script.js#L1-L235

I checked your javascript file but couldn't find an array of objects with information for your map and popups as you can see above :arrow_up: , please make the necessary changes